urllib3/_collections.py

```diff
@@ -278,10 +268,7 @@ def __repr__(self):
     def _copy_from(self, other):
         for key in other:
             val = other.getlist(key)
-            if isinstance(val, list):
-                # Don't need to convert tuples
-                val = list(val)
-            self._container[key.lower()] = [key] + val
+            self._container[key.lower()] = [key] + list(val)
```

Comment:
I think here `list(val)` will just be fine. [getlist](https://github.com/yetingsky/urllib3/blob/master/urllib3/_collections.py#L257) will always return list, according to docstring and implementation.
Actually, `isinstance(val, list)` may never reach